frontend/public/components/notification-drawer.tsx

```diff
@@ -421,6 +411,16 @@ export const ConnectedNotificationDrawer_: React.FC<ConnectedNotificationDrawerP
       </NotificationCategory>
     ) : null;
 
+  const clusterID = getClusterID(useClusterVersion());
+  if (useShowServiceLevelNotifications(clusterID)) {
```

Comment:
nit: I'm a fan of keeping all the hooks at the top of the function outside of any conditionals since they always need to be called in the same order every render. There are a couple hundred lines before this hook is called, and it'd be easy for someone to accidentally add an early return before this block in a future PR. Even if it's not strictly an issue now, it's a way to code defensively to avoid future bugs.

Reply:
You are right - I hadn't done another review of the code before making the PR.  Wanted to ensure that the tests were passing.  Should be fixed now.